### ISSUE TITLE
generating ASTs: use flux bin method going forward?

### ISSUE THREAD

**Issue by lea-hagen** [BEAST-Fitting/beast#327]
After discussion in #308 and conversation at last week's BEAST hack day, there is a preference for using the flux bin method to select artificial stars.  The next question is whether we should remove the code that does the random selection method.  If we're not going to use it, there's no reason to keep it around (though thanks to git, we can always find it again if needed).

@benw1, @galaxyumi, @lcjohnso: do you have thoughts?

**Comment by benw1** [comment]
I'm a big fan of having options.  The new default should be flux bins, 
but since we have the ability, we could allow an option to set it to 
random.  Just my preference though.

**Comment by karllark** [comment]
@benw1 : I agree with you that it is nice to have options, but that comes with the burden of having to maintain/test the code.  To be more specific, do you think you will use the random selection method based on stepping through age?  We can always get the code back if we need to (git!).  But if you do not think you'll use this option, that is useful input for this decision.

**Comment by drvdputt** [comment]
Hi everyone. I just wanted to contribute a plot to this discussion, which will be going into my IC63 paper. The mag cuts are at 1 mag above the faintest (90th percentile), and 1 mag below the brightest sources from the catalog. (I have 7 bands in total, but I'm only showing three here).
![histogram](https://user-images.githubusercontent.com/7421197/57935654-9462c400-78c2-11e9-89be-6b6dddf739c0.png)

**Comment by karllark** [comment]
Sounds like keeping the random picking of stars is good to have as an option, but not as the default or recommended.  @lea-hagen & @benw1 do you both agree?

**Comment by lea-hagen** [comment]
That seems reasonable to me.
